src/hotspot/share/prims/jvmtiRedefineClasses.hpp

```diff
@@ -302,7 +302,7 @@
 //
 // - How do we serialize the RedefineClasses() API without deadlocking?
 //
-// - SystemDictionary::parse_stream() was called with a NULL protection
+// - KlassFactory::create_from_stream() was called with a NULL protection
```

Comment:
Maybe delete the comment that goes from lines 305 - 309 ?

Reply:
Good idea.  The comment is really old and no longer relevant.